qemu/tests/getfd.py

```diff
@@ -52,14 +52,14 @@ def has_fd(pid, filepath):
         path = os.path.join(fdfiles_dir, name)
         fd = os.open(path, os.O_RDWR | os.O_CREAT)
         response = vm.monitor.getfd(fd, name)
-        os.close(fd)
```

Comment:
Keeping this line is the better choice, because:
1. file descriptors in avocado process is no longer needed.
2. to avoid the "too many open files" issue.